contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -120,47 +70,43 @@ def _clone_repo(repo_url, verifyssl=True, branch='master'):
         Repo.clone_from(repo_url, abs_local_path, branch=branch, depth=1)
         return abs_local_path
 
-    def _move_packs(self, abs_repo_base, packs, abs_local_path, subtree):
+    def _move_pack(self, abs_repo_base, pack_name, abs_local_path):
```

Comment:
> The logic remains "clone to temp, move to opt/stackstorm/packs? If so, why? why not clone to the location?

From what I can tell, we were cloning to temp location to be able to work with subtree. Since we don't have a subtree option now, we should indeed clobne to packs folder directly, but this would mean you can't install the same pack twice (think of git error 'directory is not empty' on clone). Which may not actually be a bad thing considering there is no obvious way to go about it.